### ISSUE TITLE
Translating to Brazilian Portuguese

### ISSUE THREAD

**Issue by cogumm** [LewisVo/Awesome-Linux-Software#231]
Criei este issue para tentar organizar as traduções para o [Português do Brasil](https://github.com/LewisVo/Awesome-Linux-Software/blob/master/README_pt-BR.md).

A sua ajuda é muito bem vinda.
Abraços!

**Comment by cogumm** [comment]
Podemos usar este tópico para discutir as traduções?
@FannyVieira @rodrigogirao

**Comment by LewisVo** [comment]
Sim. Isso é incrível. Continue com o bom trabalho. Além disso, a lista parece estar em estado estável agora. Não haverá grandes mudanças, em comparação com a semana passada.
p/s: I use google translate :sweat_smile:

**Comment by nyctophilio** [comment]
i'm sorry. i  haven't done any contribution yet but i'm not going to
contribute, i just want to learn from it. Its happen that i accidentally
click on pull request.

P/s: i'm no
Brazilian Portuguese😂😂😂

2017-09-04 22:18 GMT-07:00 Ramil Berces <bercesr@gmail.com>:

**Comment by nyctophilio** [comment]
despacito~

**Comment by edsoncelio** [comment]
Olá, em breve vou começar a contribuir com as traduções também, vou acompanhar o andamento por aqui também :+1:

**Comment by cogumm** [comment]
O que acham da tradução de **Awesome** ?

**Comment by edsoncelio** [comment]
Então, em algumas listas que acompanho, mesmo em ptbr elas estão com o termo **Awesome**, não sei se existe um consenso quanto a isso, mas todas as traduções aproximadas do termo acabam parecendo um pouco deslocadas, então concordaria pela não tradução do termo. Fico no aguardo de mais opiniões para que possamos formar um consenso.

**Comment by FannyVieira** [comment]
Claro! Podemos dividir as seções que cada um pode ficar responsável @cogumm

**Comment by LewisVo** [comment]
To make easier for you to track the changes (without needing to go through all commits) and assign tasks, I have an idea. I made a google spreadsheet for this https://docs.google.com/spreadsheets/d/1OwpT9A2X1FP0BEd0CGS9iFaysjELvXiujPcvkGQB2G8/edit?usp=sharing 
This is where every translation part will be assigned and if I update some category in the future, I will leave a mark there so you just have to come back monthy to check and keep your part updated(if you want) without going through github commits.
How do you guys feel about this? Right now I cannot share with edit permission but feel free to request me.

**Comment by cogumm** [comment]
@LewisVo  interesting this worksheet, let's wait for others to respond or give their opinions.

**Comment by cogumm** [comment]
I'm intending to go back to work, is anyone still looking for a translation?

**Comment by JoseRenan** [comment]
@cogumm sobre a tradução de **Awesome**, eu concordo com o @edsoncelio e acho que o mais correto é manter o Awesome, também por ser uma palavra que, nesse contexto, não tem a intenção de passar o sentido literal, mas é meio que um tipo de identificador pro tipo de repositório.

**Comment by FannyVieira** [comment]
Exatamente @JoseRenan
